rr_iarrc/src/laplacian_line_detection/laplacian_line_detector.cpp

```diff
@@ -95,45 +101,43 @@ cv::Mat kernel(int x, int y) {
 cv::Mat fillColorLines(cv::Mat lines, cv::Mat color_found) {
```

Comment:
I __think__ OpenCV will try its best to remove unnecessary copying of data from within its Mat class. However, best practice dictates using const references (const LargeObject& my_param) to minimize copying. Often times, writing fast C++ code is a matter of avoiding copying data in memory.